src/style.css

```
@@ -0,0 +1,39 @@
+@import url(./card.css);
+@import url(./star.css);
+
+@font-face {
+  font-family: Roboto;
+  src: url(fonts/Roboto-Regular.ttf);
+  font-style: normal;
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: Roboton;
+  src: url(fonts/Roboto-Medium.ttf);
+  font-style: normal;
+  font-weight: 600;
+}
+
+@font-face {
+  font-family: Robotos;
+  src: url(fonts/Roboto-Bold.ttf);
+  font-style: normal;
+  font-weight: 700;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+h1 p {
+  margin: 0;
+}
```

[DRY](https://www.plutora.com/blog/understanding-the-dry-dont-repeat-yourself-principle)

```suggestion
body,
h1,
p {
  margin: 0;
}
```